pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -480,6 +480,10 @@ def reconciliate_processes(self, trace: str):
 
         sync_dir = self._sync_dir
 
+        if not sync_dir:
+            rank_zero_warn("pid is not shared between processes in DDP and sync_dir is not initialized")
```

Comment:
Can we improve the warning message for users who have no idea about DDP or deadlocks or PIDs?

Reply:
```suggestion
            rank_zero_warn("Error handling mechanism for deadlock detection is uninitialized. Skipping check.")
```
?